### PR TITLE
Enhance page titles for SEO

### LINKED ISSUE #193: Page titles
* Can we use `-` as a separator instead of `|`? For example `Architecture - JSON Forms` instead of `Architecture | JSON Forms`
* Please check all page titles whether they make sense and adjust accordingly
  * For example the support page has no proper title
  * For example the `Basic | JSON Forms` title should be changed to `Basic examples - JSON Forms` so it's self explanatory

Motiviation: Some tools, like our new Discourse Forum, use these titles as the text for links. They should look proper. Also they are obviously shown in Browser Tabs ;)

### PR DESCRIPTION
**Pull request by TheZoker**
- Uses `-` as page title separator
- Fix support page title
- Add "Example" to page title (within examples)
- Remove unnecessary `controls.js` file
- Rename `layoutexample.mdx` to `layout.mdx` since it is already in the example folder
- Add blog title, but this is ignored in the current version of docusaurus. So an update of docusaurus is needed to fix this issue.

Closes #193

### CODE REVIEW

File: content/pages/index.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ function Home() {
   const {siteConfig = {}} = context;
   return (
     <Layout
-      title={siteConfig.title}
+      title={siteConfig.tagline}
```
Comment (sdirix) [inline content/pages/index.js:40]: What does this change do exactly?

File: content/docs/uischema/controls.mdx [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ---
 id: controls
 title: Controls
+description: Controls represent the basic building blocks for creating forms
```
Comment (sdirix) [inline content/docs/uischema/controls.mdx]:
```suggestion
description: Controls generally represent properties declared in JSON Schema and serve to manipulate data.
```

**Review by sdirix — CHANGES_REQUESTED** [review]
I like it! Only minor comments

File: content/pages/index.js (hunk 1 above)
Comment (TheZoker) [reply content/pages/index.js:40]: The index page title is `More forms. Less code. - JSON Forms` instead of `JSON Forms | JSON Forms`

**Review by sdirix — APPROVED** [review]

**Pull request merged**